maat-court-data-api/src/main/java/gov/uk/courtdata/eform/service/EformsDecisionHistoryService.java

```diff
@@ -36,7 +36,11 @@ public EformsDecisionHistory getNewEformsDecisionHistoryRecord(Integer usn) {
 
     @Transactional(readOnly = true)
     public EformsDecisionHistory getPreviousEformsDecisionHistoryRecordWroteToResult(Integer usn) {
-        return eformsDecisionHistoryRepository.findFirstByUsnAndWroteToResultsOrderByIdDesc(usn, WROTE_TO_RESULT);
+        try {
+            return eformsDecisionHistoryRepository.findFirstByUsnAndWroteToResultsOrderByIdDesc(usn, WROTE_TO_RESULT);
+        }catch (Exception e){
+            return new EformsDecisionHistory();
+        }
```

Comment:
Please add the same as EformAuditService changes  
Example : 
eformsDecisionHistoryRepository.findFirstByUsnAndWroteToResultsOrderByIdDesc(usn, WROTE_TO_RESULT)
                .orElse(EformsDecisionHistory.builder().build());